### ISSUE TITLE
base_radius

### ISSUE THREAD

**Issue by ghost** [RVLVR/REVClusterMap#9]
Maybe I'm wrong here, but I thought that if you decrease BASE_RADIUS I could see more annotations.
However BASE_RADIUS and MINIMUM_LATITUDE_DELTA are not used anywhere. 
Everything else seems great!

**Comment by filipandrei87** [comment]
@toydor did you figure out how to change the base radius?

**Comment by ghost** [comment]
No, I never really understood how to do it. I used my own solution without Revolver. Hope you understand it better than me.

**Comment by RVLVR** [comment]
It doesn't use the BASE_RADIUS to determine the clustering ( i think its a lost property in my code )

they way it works it uses the definition BLOCKS to determine on how small or how big you want clustering to happen.

it uses the visibleRect and divides it by BLOCKS definition

for example if BLOCKS property is 4

it divides the screen in 16 equal blocks and all pin's within that block get clustered, if you want to have less clusters just pick a higher number for the BLOCKS property so that there are more blocks created.

but the manier blocks created the more calculation is needed so don't pick a to high number.
